editor/store/reducer.js

```diff
@@ -485,6 +485,60 @@ export function blocksMode( state = {}, action ) {
 	return state;
 }
 
+/**
+ * Reducer returning the currently autosaving status.
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} action Dispatched action
+ * @return {Object}        Updated state
+ */
+export function currentlyAutosaving( state = false, action ) {
+	switch ( action.type ) {
+		case 'DOING_AUTOSAVE':
+			const { isAutosaving } = action;
+			return isAutosaving;
+	}
+
+	return state;
+}
+
+/**
+ * Reducer returning the currently autosaving status.
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} action Dispatched action
+ * @return {Object}        Updated state
+ */
+export const autosave = withChangeDetection( function( state = false, action ) {
```

Comment:
yes, that is how i am using it, i am tracking the autosave changes using this wrapper. check the updated code from today which is a little more refined. I kept autosave outside editor in the reducer because it was cleaner that way, could possible be combined.